_projects/2025/100528259/100528259.Rmd

```diff
@@ -0,0 +1,986 @@
+---
+title: "An 8-Bit Look at the Evolution of BMI Values of Olympic Athletes"
+description: |
+  This study displays the evolution of BMI values of Olympic athletes who competed in the Summer Olympic Games of 1960 and 2016.
+categories: "2024"
+author: Gür Piren
+date: "`r Sys.Date()`"
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(out.width="100%", fig.align="center")
+```
+
+## Packages and fonts
+
+```{r, echo=TRUE}
+
+# For replication graph
+
+library(tidyverse)
+library(readr)
+library(ggplot2)
+library(dplyr)
+library(ggrepel)
+library(extrafont)
+library(showtext)
+library(sysfonts)
+
+# Extras - For alternative plot
+
+library(grid)
+library(ggtext)
+library(gridExtra)
+
+
+# Fonts for the replication plot 
+
+font_add_google("Crimson Pro", "crimson-pro")
+font_add_google("Outfit", "outfit")
+font_add_google("Cormorant Garamond", "cormorant-garamond")
+font_add_google("Montserrat", "Montserrat")
+font_add_google("Archivo Black", "ArchivoBlack")
+font_add_google("Piazzolla", "PiazzollaExtraLight")
+
+# Fonts for the improved plot
+
+font_add_google("Press Start 2P", "press-start")
+font_add_google("VT323", "vt323") 
+font_add_google("DotGothic16", "dotgothic16")
+
+showtext_auto()
+
+```
+
+The data set displays personal records of Olympic athletes.
+
+The original plot focuses on athletes who are at least 20 years old and competed in 1960 and 2016 Summer Olympic games. Taking into account their height and weight values, it then calculates BMI values of athletes and categorise them.
+
+In so doing, it creates four different facets and coloured tiles in each one of them. Obviously, each colour tile represents a BMI category.
+
+The four facets are designed to represent 1960 and 2016 Summer Olympic games, and male and female athletes.
+
+## Replication 
+
+### Working with the data
+
+Reading the dataset
+
+```{r, echo=TRUE}
+
+data_main <- read_csv("/Users/gpen/Dropbox/Mac/Desktop/Version Control/dataviz_gur_piren/athlete_events.csv")
```

Comment:
Please include this CSV in this PR close to this Rmd. Then, you can read the file just with the name, removing any path from your computer.